app/views/courses/index.html.haml

```diff
@@ -20,6 +21,7 @@
       %td= course.start_date
       %td= course.end_date
       %td= course.allocation
+      %td= course&.institute&.name
```

Comment:
Looking at the other methods called, you won't need the safe operator for accessing institute, since `course` is proven not to be nil based on previous calls

```suggestion
      %td= course.institute&.name
```